os_builders/roles/nubes_bootcontext/tasks/update_cloud_users.yml

```diff
@@ -0,0 +1,31 @@
+
+- name: Install jq on Ubuntu
+  block:
+    - name: Install jq for Pakiti
+      apt:
+        name: "jq"
+        state: present
+        update_cache: yes
+  when: ansible_distribution == "Ubuntu"
+
+- name: Install jq on RL
+  yum:
+    name: "jq"
+    state: present
+  when: ansible_distribution == "Rocky"
```

Comment:
I think these can both be done generically with the `package` module https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_module.html
```
- name: Install jq
  ansible.builtin.package:
    name: jq
    state: present
```

(Since an earlier step will update_cache to update the cloud image to use the latest packages)

Reply:
Done